internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -61,6 +62,12 @@ func resourceLogAnalyticsWorkspace() *pluginsdk.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
 
+			"allow_resource_only_permissions": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
This shouldn't be Computed since this would hide the default value of false if unset. Can we just set `Default: false` here instead?